src/Controls/samples/Controls.Sample/Pages/XamlPage.xaml

```diff
@@ -1,13 +1,30 @@
 ﻿<controls:BasePage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
                    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
                    xmlns:controls="clr-namespace:Maui.Controls.Sample.Controls"
+                   xmlns:layout="clr-namespace:Microsoft.Maui.Controls.Layout2;assembly=Microsoft.Maui.Controls"
```

Comment:
Should we be putting these layouts into the same xmlns as all the others?

Reply:
I don't think we can right now, because the type names would conflict. `GridLayout` would be fine, but the V/HStack classes have a base class called StackLayout, and I think it would conflict.